.github/workflows/unit-tests.yml

```
@@ -22,11 +22,7 @@ jobs:
                       can-fail: false
                     # Development versions
                     - php: '8.1-rc'
```

```suggestion
                    - php: '8.1'
```
While at it, would you mind changing that? It's stable now

---

I will add it to the php version list on line 13 ;)

---

I have rebased it on master and added symfony 6.0 for php 8.0 and php 8.1 only.

---

@chalasr The job matrix update is already proposed in #67

---

@chalasr It would probably be best to merge https://github.com/thephpleague/oauth2-server-bundle/pull/67 first. That PR should take care of the CI.